addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1478,6 +1478,8 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
 
         this.colorPreviewEl.style.removeProperty('background-color');
         this.colorPreviewEl.style.removeProperty('background-image');
+        this.colorPreviewEl.className = Array.from(this.colorPreviewEl.classList)
+                .filter(c => !c.startsWith('bg-')).join(' ');
```

Comment:
Note that we have the same issue with the preview of the text bg color in the editor toolbar. Maybe it needs the same kind of fix ...

Reply:
Commit https://github.com/odoo/odoo/commit/212a8bfdd21269b18054200b9e2585e1c95540d6#diff-ffb61e86e6b8297ef8997f9c605de14de896b54161c032d88a80bbaeac4f89beL1484 removed the working line, we should just restore it.